frontend/src/components/pages/MagazineReview/FeaturedReview.tsx

```diff
@@ -0,0 +1,233 @@
+        </chakra.div>
+        <Image
+          h="100%"
+          w="100%"
+          flexGrow={1}
+          maxHeight="320px"
+          src="https://upload.wikimedia.org/wikipedia/en/1/10/The_Cat_in_the_Hat.png"
+          fit="cover"
+        />
+      </chakra.div>
+    </Box>
+  );
+};
+
+const MobileFeaturedReview = (): JSX.Element => {
+  return (
+    <chakra.div
+      h="200px"
+      w="216px"
+      display="flex"
+      boxShadow="xl"
+      borderRadius="5px"
+      overflow="hidden"
+    >
+      <chakra.div
+        h="100%"
+        display="flex"
+        flexDirection="column"
+        justifyContent="center"
+        px={2}
+        bg="white"
+      >
+        <Text fontSize={20} fontWeight={500}>
+          name
+        </Text>
+        <Text fontSize={16}>author</Text>
+        <Text color="gray" fontSIze={16}>
+          yyyy-mm-dd
+        </Text>
+
+        <Box display="inline" fontSize={16}>
+          genre
+        </Box>
+      </chakra.div>
+      <Image
+        h="100%"
+        w="100%"
+        flexGrow={2}
+        maxHeight="320px"
+        src="https://upload.wikimedia.org/wikipedia/en/1/10/The_Cat_in_the_Hat.png"
+        fit="cover"
+      />
+    </chakra.div>
+  );
+};
+
+const CategoryReviews = ({
+  name,
+  link,
+  reviews,
+}: CategoryReviewsProps): React.ReactElement => {
+  const history = useHistory();
+  const displayReviews = useBreakpointValue(
+    {
+      base: 2,
+      lg: 5,
+      md: 3,
+      sm: 3,
+    },
+    "lg",
+  );
+
+  const [toTranslate, setToTranslate] = useState(0);
+  const prevRef = useRef<HTMLButtonElement>(null);
+  const nextRef = useRef<HTMLButtonElement>(null);
+
+  const lengthCarousel = reviews.length;
+
+  const updateIndex = (newIndex: number) => {
+    let index = newIndex;
+    if (newIndex < -1) {
+      index = -1;
+    } else if (newIndex >= lengthCarousel - 1) {
+      index = lengthCarousel - 2;
+    }
+    setToTranslate(index);
+  };
+
+  const [windowWidth, setWindowWidth] = useState(window.innerWidth);
+  useEffect(() => {
+    function handleResize() {
+      setWindowWidth(window.innerWidth);
+    }
+
+    window.addEventListener("resize", handleResize);
+    return () => window.removeEventListener("resize", handleResize);
+  }, []);
+
+  return (
+    <Box mt="20px" maxW="100vw">
+      {reviews.length > 0 ? (
+        windowWidth < 768 ? (
+          <MobileFeaturedReview />
+        ) : (
+          <Flex gap={20} alignItems="center">
+            <button
+              type="button"
+              ref={prevRef}
+              onClick={() => updateIndex(toTranslate - 1)}
+            >
+              <ChevronLeftIcon h={50} w={50} cursor="pointer" />
+            </button>
+            <Box
+              overflow="hidden"
+              w={{ base: "300px", md: "600px", lg: "900px" }}
+              h={{ base: "300px", md: "300px", lg: "420px" }}
+              pt="30px"
+            >
+              <chakra.div
```

Comment:
Why are we using `chakra.div` and not `Box`?

Reply:
I forget to be honest. Everything works with Box so I'll roll with that